stack/app/variables.tf

```
@@ -59,11 +59,11 @@ variable "private_buckets_list" {
     bucket_name                     = string
     versioning                      = bool
     primary_storage_class_retention = number
-    replicas = list(object({
+    replicas = optional(list(object({
       bucket_arn = string
       kms_arn    = string
       region     = string
-    }))
+    })))
```

```suggestion
    })), [])
```

does this work to have a default, so we do not need to compare to null everywhere? https://developer.hashicorp.com/terraform/language/expressions/type-constraints#optional-object-type-attributes